netweaver/install_aas.sls

```diff
@@ -63,7 +65,7 @@ netweaver_install_{{ instance_name }}:
     - name: {{ node.sid.lower() }}
     - inst: {{ instance }}
     - password: {{ netweaver.sid_adm_password|default(netweaver.master_password) }}
-    - software_path: {{ netweaver.swpm_folder|default(netweaver.swpm_extract_dir) }}
+    - software_path: {{ netweaver.swpm_folder|default(swpm_extract_dir) }}
```

Comment:
I would put the value in `defaults.yaml` and that's it

Reply:
@arbulu89 `swpm_extract_dir` is based on pillar entry : `nw_extract_dir'~'/SWPM`
Defining in `defaults.yaml` sets a fixed value for it

I want to have the nw media extracted to subfolders under a user defined location(`nw_extract_dir`).